src/labels/CustomLabels.labels

```diff
@@ -2727,6 +2783,22 @@
         <shortDescription>Validation and health check solution when status is not mapped to a state.</shortDescription>
         <value>Go to NPSP Settings | Recurring Donations | Status to State Mappings, or contact your administrator for help.</value>
     </labels>
+    <labels>
+        <fullName>RD2_StatusMustBeActiveOrPause</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Pause request requires Active or Paused status</shortDescription>
+        <value>A pause request requires a Recurring Donation in Active or Paused status.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_UnPauseRequestCannotIncludeDates</fullName>
+        <categories>Recurring-Donations, Pause</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>UnPause requests cannot include dates</shortDescription>
+        <value>UnPause requests cannot include Start Date or End Date.</value>
```

Comment:
@andrewyu-salesforce - How is "UnPause" styled in the API request? "unPause" or "UnPause"?

Reply:
It is styled `unPause` in our code, however the callable API is not case sensitive so UnPause would be valid to use.